src/runtime/app.ts

```diff
@@ -117,7 +119,8 @@ export function create(): App {
     assemble() {
       if (appState.assembled) return
 
-      appState.assembled = {} as any
+      // todo https://github.com/graphql-nexus/nexus/pull/788#discussion_r420645846
+      appState.assembled = {} as Partial<AppState['assembled']>
```

Comment:
This tweak made all tests failed

Reply:
Yeah this wasn't supposed to get through :(